src/docker/compose.dss-local.yaml

```diff
@@ -2,6 +2,18 @@ version: "3.9"
 
 services:
 
+  streamer:
+    privileged: true
+    build:
+      context: ../../
+      dockerfile: src/docker/tss/Dockerfile
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:18084/readiness"]
```

Comment:
This doesn't work yet: it's 404 despite Aptos' advisement (see chat with them). They're aware and looking into it.
@CRBl69 If you can find another way to health check please do.

Reply:
![image](https://github.com/econia-labs/econia/assets/67070716/a919b82b-337b-4e64-bdd4-dee7cc322a20)

Seems like it works.

Also, Docker docs say this:

![image](https://github.com/econia-labs/econia/assets/67070716/f7162555-96cc-4639-836c-1e53915e1eca)

When `curl` gets a 404 it seems to exit with code 22:

![image](https://github.com/econia-labs/econia/assets/67070716/f88a3f23-c195-4ed4-97cb-6487a7af3e47)

I think you should put `test: ["CMD-SHELL", "curl -f http://localhost:18084/readiness || exit 1"]` here.